providers/mondoo/resources/mondoo_space.go

```diff
@@ -5,14 +5,32 @@ package resources
 
 import (
 	"context"
+	"errors"
 	"time"
 
 	"go.mondoo.com/cnquery/v11/llx"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/plugin"
 	"go.mondoo.com/cnquery/v11/providers/mondoo/connection"
 	"go.mondoo.com/cnquery/v11/types"
 	mondoogql "go.mondoo.com/mondoo-go"
 )
 
+func initMondooSpace(runtime *plugin.Runtime, args map[string]*llx.RawData) (map[string]*llx.RawData, plugin.Resource, error) {
+	conn := runtime.Connection.(*connection.Connection)
+	if conn.Type != connection.ConnTypeSpace {
+		return nil, nil, errors.New("cannot initialize mondoo.space, invalid connection")
+	}
+	args["mrn"] = llx.StringData(conn.Upstream.SpaceMrn)
+	// TODO: we should fetch this from the API, mrn basename != name
+	args["name"] = llx.StringData(connection.MrnBasenameOrMrn(conn.Upstream.SpaceMrn))
```

Comment:
I think we should turn that into a resolver to fetch the space details.

Reply:
+1, thats the TODO above that line

lets do it in a follow-up PR